notebook2/notebook/notebook.py

```diff
@@ -370,10 +387,9 @@ def notebook_post():
 
     jupyter_token = uuid.uuid4().hex
     name = request.form.get('name', 'a_notebook')
-    safe_user_id = user_id_transform(g.user['auth0_id'])
+    safe_id = user_id_transform(g.user['auth0_id'])
```

Comment:
name changed to fit in line ending 79 char constraint. prepping for future flake8 linting